fastify.js

```diff
@@ -128,6 +128,7 @@ function fastify (options) {
 
   // Instance Fastify components
   const { logger, hasLogger } = createLogger(options)
+  const keepAliveConnections = new Set()
```

Comment:
Should the weakset fit this use case?
It should be doable ignoring the socket on close event too

Reply:
No. It is not iterable.